libs/tailwindcss-config/src/vega-custom-classes.js

```diff
@@ -34,6 +34,9 @@ const vegaCustomClasses = plugin(function ({ addUtilities }) {
     '.clip-path-rounded': {
       clipPath: 'circle(50%)',
     },
+    '.color-scheme-dark': {
+      colorScheme: 'dark',
```

Comment:
question (non blocking): Not familiar with this css property. Whats it for

Reply:
It allows you to tell a component to render internal elements in dark mode. We use it on inputs, this allows things like the datepicker internals to render with the correct colour. 

In Tailwind dark mode, without the property:

<img width="406" alt="Screenshot 2022-09-14 at 14 19 55" src="https://user-images.githubusercontent.com/2410498/190165689-3ba1e4df-2f3b-4722-befa-adb2758f2457.png">

With:

<img width="403" alt="Screenshot 2022-09-14 at 14 20 16" src="https://user-images.githubusercontent.com/2410498/190165711-026df379-7de1-4750-9f0d-c26a15820473.png">